code/modules/mob/living/carbon/human/species_types/klingoffs_elves.dm

```diff
@@ -0,0 +1,15 @@
+/datum/species/klingoff
+	name = "Klingoff"
+	id = "klingoff"
+	default_color = "FFFFFF"
+	specflags = list(EYECOLOR,HAIR,FACEHAIR,LIPS)
+	mutant_bodyparts = list()
+	default_features = list()
+
+/datum/species/space_elf
```

Comment:
These should be mutant parts if they're just ears

Reply:
Reasons this is dumb:

I'm sacrificing the unique species aspect
I cant make them forced vegetarian
I have to hardcode mutant part checks for shit with elves vs catbeest retards

Reply:
before you say "make the ears do the effects"

1. why would I want ears to make you vegetarian
2. i refuse to allow elves with cat tails

Reply:
1. surprisingly principled of you considering how you normally work.
2. we can actually do that with surgery